src/lib.rs

```diff
@@ -89,6 +87,8 @@
 //! - `multiple-pymethods`: Enables the use of multiple [`#[pymethods]`](macro@crate::pymethods)
 //! blocks per [`#[pyclass]`](macro@crate::pyclass). This adds a dependency on the [inventory]
 //! crate, which is not supported on all platforms.
+//! - `pyproto`: Enables the deprecated `#[pyproto]` attribute macro. This will be removed in a 
+//! future release.
```

Comment:
Do we want to mention a concrete version? 0.18 perhaps?

Reply:
I think it's fine to say 0.18 here 👍 (I'm already planning to have a bonfire PR as soon as 0.17 is out the door xD)